etc/LATISS/primary_hdu.header

```diff
@@ -91,24 +91,9 @@ IMAGETAG=                      / DAQ Image id
 OBSANNOT=                      / Observing annotation
 
         ---- Information from Camera ----
-CCD_MANU=                      / CCD Manufacturer
 CCD_TYPE=                      / CCD Model Number
-CCD_SERN=                      / Manufacturers CCD Serial Number
-LSST_NUM=                      / LSST Assigned CCD Number
-SEQCKSUM=                      / Checksum of Sequencer
-SEQNAME =                      / SequenceName from Camera StartIntergration
-REBNAME =                      / Name of the REB
-CONTNUM =                      / CCD Controller (WREB) Serial Number
 TEMP_SET=                      / Temperature set point (deg C)
```

Comment:
This keyword and the next are in ComCam's sensor HDU, why not put them in LATISS's?

Reply:
For LATISS, `TEMP_SET` comes a scalar from the telemetry in `ATCamera.fp_Ccd.temp`, so it stays on the common `PRIMARY` header template.